addons/web/static/src/js/services/crash_manager.js

```diff
@@ -122,6 +124,18 @@ var CrashManager = AbstractService.extend({
                     window.onOriginError();
                     delete window.onOriginError;
                 } else {
+                    // In Safari 16.4+ (as of Jun 14th 2023), an error occurs
+                    // when going back and forward through the browser when the
+                    // cache is enabled. A feedback has been reported but in the
+                    // meantime, hide any script error in these versions.
+                    if (
+                        config.device.isIOS
+                        && message === "Script error."
```

Comment:
Seems useless as the famous `Script error.` means that we are in the CORS error case that we are catching just a few lines above through `if (!file && !line && !col) {` in the Chrome/Opera/Safari case.
So it's kinda redundant, but it's fine.

https://github.com/odoo/odoo/pull/21032#issuecomment-367753861

> Browser differences:
>   
> 1. Chrome & Opera: `message = 'Script Error.'`, `error = null`
> 2. Firefox [is handling it directly](https://dxr.mozilla.org/mozilla-beta/source/dom/base/nsJSEnvironment.cpp#464) and return `uncaught exception: [object Object]` instead of `Script Error.` for `message` and set `error` to the error with the message object that Chrome & Opera are blocking.
> 3. Safari: Need to check, but since it is a webkit browser, it most likely handle it like Chrome & Opera 
> 
> [Mozilla 'Script Error.' handler](https://dxr.mozilla.org/mozilla-beta/source/dom/base/nsJSEnvironment.cpp#464)
[WebKit 'Script Error.'](http://trac.webkit.org/browser/branches/chromium/648/Source/WebCore/dom/ScriptExecutionContext.cpp?rev=77122#L301)
> Coming from this [stackoverflow](https://stackoverflow.com/questions/5913978/cryptic-script-error-reported-in-javascript-in-chrome-and-firefox)
> 
> Found this too:
https://bugs.webkit.org/show_bug.cgi?id=132945
https://bugs.webkit.org/show_bug.cgi?id=70574